src/catalog.js

```diff
@@ -237,7 +239,14 @@ class Page {
   }
 
   static create({ category, parent, slug, sources }) {
-    const matter = frontMatter(sources);
+    var matter;
```

Comment:
here you set matter to sources, on row 242 you reset it, change it to one variable.

Reply:
Jag tog bort const matter = frontMatter(sources) och ersatte den med var matter. Det enda som är med är var matter.

Reply:
lets make a new pull request then